dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/TaskInstanceService.java

```diff
@@ -111,18 +116,35 @@
             return result;
         }
 
+
         Page<TaskInstance> page = new Page(pageNo, pageSize);
+        PageInfo pageInfo = new PageInfo<TaskInstance>(pageNo, pageSize);
+
+        //executor name query
+        int executorId = 0;
+        if (StringUtils.isNotEmpty(executorName)) {
```

Comment:
Can duplicate code be refactored together？
Suggest that put these them in UserService.

Reply:
thx, i will fix it